bolt/metrics.go

```diff
@@ -21,6 +21,8 @@ var (
 	scraperBucket         = []byte("scraperv2")
 	telegrafBucket        = []byte("telegrafv1")
 	telegrafPluginsBucket = []byte("telegrafPluginsv1")
+	remoteBucket          = []byte("remotesv2")
```

Comment:
Not for this PR, but I agree with `TODO: nuke this whole thing?` above - we shouldn't push the phone home metrics through boltdb.